core/src/main/java/de/danoeh/antennapod/core/storage/DBTasks.java

```diff
@@ -463,6 +463,38 @@ public static FeedItem getQueueSuccessorOfItem(final long itemId, List<FeedItem>
     }
 
     /**
+     * Returns the top of a queue
+     *
+     * @param itemId ID of FeedItem
+     * @param queue  If this parameter is null, the method will load the queue from the database in the same thread
+     * @return The top of the queue or null
```

Comment:
Just a nitpick: Maybe you want to add in which case null is returned (empty queue).